cmd/podman/varlink/io.projectatomic.podman.varlink

```diff
@@ -124,6 +124,55 @@ type ContainerNameSpace (
     ipc: string
 )
 
+# InfoHost describes the host stats portion of PodmanInfo
+type InfoHost (
+    mem_free: int,
+    mem_total: int,
+    swap_free: int,
+    swap_total: int,
+    arch: string,
+    cpus: int,
+    hostname: string,
+    kernel: string,
+    os: string,
+    uptime: string
+)
+
+# InfoGraphStatus describes the detailed status of the graphc
```

Comment:
I don't think graphc is a term we ever use. "Storage driver" or "graph driver" is a lot more in common with the terms we're actually using.

Reply:
typo